projects/ngx-vest-forms/src/lib/directives/form-model.directive.ts

```diff
@@ -26,12 +27,15 @@ import { getFormControlField } from '../utils/form-utils';
 })
 export class FormModelDirective implements AsyncValidator {
   private readonly formDirective = inject(FormDirective);
+  private readonly validationOptionsDirective = inject(ValidationOptionsDirective, { self: false, optional: true });
```

Comment:
Again, I think the self should be true, isn't this guy trying to inject the validationsOptions on this component?

Reply:
Replace the inject by an input ;)